/**
 * This file is part of the "libfnord" project
 *   Copyright (c) 2015 Paul Asmuth
 *
 * FnordMetric is free software: you can redistribute it and/or modify it under
 * the terms of the GNU General Public License v3.0. You should have received a
 * copy of the GNU General Public License along with this program. If not, see
 * <http://www.gnu.org/licenses/>.
 */
package stx.feeds;

message Message {
  required uint64 offset = 2;
  required uint64 next_offset = 3;
  required string data = 4;
  optional uint64 time = 5;
  optional string host_id = 6;
};

message MessageList {
  repeated Message messages = 1;
}
